src/components/single-page/single-page-content.tsx

```diff
@@ -1,13 +1,11 @@
 import React from "react";
-// import { ActivityLayouts, PageLayouts, isQuestion, VisibleEmbeddables, getVisibleEmbeddablesOnPage,
-//          EmbeddableSections, getLinkedPluginEmbeddable } from "../../utilities/activity-utils";
-import { ActivityLayouts, isQuestion, getLinkedPluginEmbeddable } from "../../utilities/activity-utils";
-import { Embeddable } from "../activity-page/embeddable";
+import { ActivityLayouts, isQuestion, getLinkedPluginEmbeddable, numQuestionsOnPreviousSections } from "../../utilities/activity-utils";
```

Comment:
TSLint saying we don't need to import `ActivityLayouts`,`isQuestion`, `getLinkedPluginEmbeddable`, or `ActivityLayouts` here.